src/com/goyeau/moulin/ScalaModule.test.scala

```diff
@@ -45,10 +45,6 @@ class ScalaModuleTest extends FunSuite:
       """.+\.scala-build/com/goyeau/moulin/cache/scalamoduletest/app/compile/-?\d+/dest/\.scala-build/dest_\w+-\w+/classes/main:.+"""
     assert(compile.matches(appClasspathRegex), s"$compile should match the regex '$appClasspathRegex'")
 
-  test("compile a module that fails compilation should fail"):
-    intercept[SubprocessException]:
-      `fail-compile`.compile()
-
```

Comment:
Removed since scala-cli runs `sys.exit(...)` and that kills the entire process, including the test itself.